src/App.css

```diff
@@ -5,7 +5,8 @@
 
```

Comment:
![readme](https://user-images.githubusercontent.com/93039359/202175079-03f9fbdd-8537-4da1-a0e7-e49a67b44783.PNG)

- [ ] Your Readme file is not descriptive enough 😨 Your Getting Started Section should have those commands which are mandatory to setup the application from cloning to starting the application on the browser 😊 shown below ⬇️

```
git clone git@github.com:Samuellekpor/Math-magicians.git
cd Math-magicians
npm install
npm start
```